app/workers/gitter_notificator.rb

```diff
@@ -0,0 +1,58 @@
+require 'gitter'
+require 'travis'
+
+class GitterNotificator
+  include Sidekiq::Worker
+  sidekiq_options :queue => :miq_bot_glacial, :retry => false
+
+  include Sidetiq::Schedulable
+  recurrence { minutely(10) }
+
+  include SidekiqWorkerMixin
+
+  def perform
+    init
+  end
+
+  private
+
+  def init
+    @gitter_token = Settings.travis_monitor.gitter_token
+
+    Settings.travis_monitor.branches.each do |element|
+      check(element.repo, element.branch, element.gitter_room)
```

Comment:
I think this `check` method does too many things at once. Let's take these functionalities apart:
* A method that **checks the build change status** 
* A method that **builds a message** based on the status
* A method that **sends the message** to gitter